### ISSUE TITLE
Input GT segmentation for training

### ISSUE THREAD

**Issue by MohammedSunoqrot** [AnnekeMeyer/AnisotropicMultiStreamCNN#1]
Hi,

I was thinking of using your network, but I need to retrain it for my data.
However, I am wondering how should the GT segmentation be for training?
Should it be one mask for the axial image only? Or should I provide manual segmentation for each of the axial, coronal and sagittal scans, drawn separately over their corresponding image?

Thanks

**Comment by AnnekeMeyer** [comment]
Hi Mohammed, 

the GT for this network architecture consists of only one (high resolution) volume.

So yes, you can use only the segmentation from the axial scan. But for a more precise GT, we saw that fusing the segmentations of axial, coronal and sagittal scans to one single (high resolution) volume created more reliable GTs (see section 2.4 in our paper http://www.var.ovgu.de/pub/2018_Meyer_ISBI_multi_plane_prostate_segmentation.pdf for details on the segmentation fusion). The resulting GT captured the base and apex regions of the prostate better than the segmentation from only the axial scans. 

So depending on the desired accuracy of the network in the base and apex region, you can choose between those two options.

**Comment by MohammedSunoqrot** [comment]
Thank you for the sufficient answer :)

**Comment by CinKKKyo** [comment]
> Thank you for the sufficient answer :)

Hi Sunoqrot, I also meet the problem to train the segmentation model on axial images with fused segmentation.
I saw the dim of images and labels mismatched, such as the case "0007", for image, the size is (384, 384, 19) and the spacing is (0.5, 0.5, 2.99); while the size of label is (320, 320, 106) and its spacing is (1, 1, 1). So my question is how to make them to have the same dim in size and spacing? Cause I saw the label is fused by three planar images, and its spacing is (1, 1, 1), is that mean I should resample the image to the spacing : (1, 1, 1) and then crop the image and label into (192, 192, 56)?
